dashboard/src/icons/Warning.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
```

Comment:
Please use the icon directly from the https://feathericons.com/ library - e.g. https://github.com/kubeapps/kubeapps/blob/88344aab04ff43d79544ce956c135bd7acf3ab6a/dashboard/src/components/ChartView/ChartReadme.tsx#L2